packs/feat-effects/effect-blessed-counterstrike.json

```diff
@@ -28,6 +28,7 @@
                 ],
                 "key": "Weakness",
                 "label": "PF2E.IWR.Custom.EnemyStrikes",
+                "nonDamage": true,
```

Comment:
A bit strange to classify a strike as non-damaging.

Reply:
I think of as it a weakness to the contact from being struck rather than as a weakness to the damage the strike does.